dwave/system/composites/virtual_graph.py

```diff
@@ -32,14 +37,19 @@
 from dwave.system.composites.embedding import FixedEmbeddingComposite
 from dwave.system.flux_bias_offsets import get_flux_biases
 
+from warnings import warn
```

Comment:
Python libraries get imported first.

Also, IMO it's nicer to just `import warnings` then do `warnings.warn` for a slightly cleaner namespace.